cryptol-remote-api/src/CryptolServer.hs

```diff
@@ -1,43 +1,43 @@
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NoMonomorphismRestriction #-}
```

Comment:
Why do we need this language extension?

Reply:
These helper functions have polymorphic types which will be inferred as monomorphic otherwise:

```haskell
    jsonString = JSON.String . Text.pack
    jsonPretty = jsonString . pretty
    jsonShow = jsonString . show
    jsonList = JSON.Array . Vector.fromList
```

The alternative is giving them all signatures, which I deemed unnecessary in this case.

Reply:
Won't type signatures help people who are reading the code?

Reply:
In this case I thought these functions were so small that the type signatures wouldn't add much; if you felt they'd have helped you, then that's enough evidence for me to add them of course!

Reply:
Resolved in 8ba57a0 in favor of using explicit type signatures.